### PR TITLE
local development environment clones the ansible repo and uses the same version as RHEL

### PR DESCRIPTION
**Pull request by yithian**

### CODE REVIEW

File: docker/local_development/start-local-dev-env.sh [hunk 1]
```diff
@@ -4,6 +4,20 @@ RED="$(echo -e '\033[1;31m')"
...
 NORM="$(echo -e '\033[0m')"
 
+# use the correct version of ansible
+# olde = openshift local dev environment
+LOCAL_DEV_ENV_ANSIBLE_DIR='/tmp/olde-ansible/'
+ANSIBLE_VERSION="v2.4.3.0-1"
+echo "Using Ansible ${ANSIBLE_VERSION}."
+
+if [ ! -d "$LOCAL_DEV_ENV_ANSIBLE_DIR" ]; then
+  git clone https://github.com/ansible/ansible.git "$LOCAL_DEV_ENV_ANSIBLE_DIR"
```
Comment (lisa) [inline docker/local_development/start-local-dev-env.sh]: is it worth doing `git clone -b $ANSIBLE_VERSION https://github.com/ansible/ansible.git` to check out the version we want during the cloning process?

Comment (yithian) [reply docker/local_development/start-local-dev-env.sh]: Yeah, that's probably worth doing, it'll save a step and makes for cleaner code.

**Comment by rogbas** [comment]
lgtm

**Review by rogbas — APPROVED** [review]
lgtm

**Pull request merged**